### PR TITLE
Add to guide

### PR DESCRIPTION
**Pull request by noeladd**

### CODE REVIEW

File: browser/js/common/directives/add-to-guide/add-to-guide.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<md-button class = "md-fab md-accent" aria-label="Add to Guide" ng-click="showAdvanced()">
+    <md-tooltip>Add Resource to Guide</md-tooltip>
+    <i class = "fa fa-plus" aria-hidden = "true"></i>
+</md-button>
+
+
+
```
Comment (masungwon) [inline browser/js/common/directives/add-to-guide/add-to-guide.html]: extra space

File: browser/js/common/directives/add-to-guide/add-to-guide.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                <form name = "guideForm">
+                    <md-input-container class = "md-block" flex-gt-sm>
+                        <label>Guide</label>
+                        <md-select ng-model="guide.id">
+                            <md-option ng-repeat = "guide in userGuides" value="{{guide.id}}">
+                                {{guide.title}}
+                            </md-option>
+                        </md-select>
+                    </md-input-container>
+
```
Comment (masungwon) [inline browser/js/common/directives/add-to-guide/add-to-guide.html:20]: extra space

File: browser/js/common/directives/add-to-guide/add-to-guide.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+					scope.guideForm.$setUntouched();
+					scope.guide = {tags: []}
+				}
+
+				scope.submitForm = function(){
+					if (scope.guide.id){
+						return GuideFactory.addResource(scope.guide.id, scope.resource)
+					}
+					else if (scope.guide.title){
+						console.log("Guide: Tags? ", scope.guide.tags);
```
Comment (masungwon) [inline browser/js/common/directives/add-to-guide/add-to-guide.js]: remove console log

File: browser/js/common/directives/add-to-guide/add-to-guide.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+app.directive('addToGuide', function($mdDialog, $mdToast, GuideFactory, $log){
+	return {
+			restrict: 'E',
```
Comment (masungwon) [inline browser/js/common/directives/add-to-guide/add-to-guide.js]: the indentations seem excessive

File: browser/js/common/directives/resource-card/resource-card.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (masungwon) [inline browser/js/common/directives/resource-card/resource-card.js:1]: extra space

File: browser/js/common/directives/resource-card/resource-card.js [hunk 6]
```diff
@@ -54,6 +55,6 @@ app.directive('resourceCard', function ($state, ResourceFactory, GuideFactory) {
 					GuideFactory.removeResource(scope.guide.id, {id: id});
```
Comment (masungwon) [inline browser/js/common/directives/resource-card/resource-card.js]: indentation

File: browser/js/guide_detail/guide.js [hunk 7]
```diff
@@ -5,10 +5,12 @@ app.config(function($stateProvider) {
         controller: 'GuideCtrl',
         resolve: {
             guide: function(GuideFactory, $stateParams){
+                console.log("In guide resolve!")
```
Comment (masungwon) [inline browser/js/guide_detail/guide.js:8]: remove console.log

**Review by masungwon — CHANGES_REQUESTED** [review]
Great job Rachel! Just remove some of the console logs and extra spaces & indentations!

**Comment by noeladd** [comment]
Fixed Console.log and Indentation Issues

**Pull request merged**